README.md

```diff
@@ -112,6 +112,12 @@ implementation "com.gojuno.koptional:koptional:$koptional_version"
 implementation "com.gojuno.koptional:koptional-rxjava2-extensions:$koptional_version"
 ```
 
+### [RxJava 3][rxjava3] Extensions
```

Comment:
This will not work since it requires a special footnote with a link (please take a look at the bottom of the file).

Reply:
Also — can I ask you to rename the _RxJava 2 Extensions_ section above to _RxJava Extensions_ if the code is the same for both? A unique one will be needed if it isn’t.